text/0000-cfg-logical-ops.md

```diff
@@ -20,13 +20,46 @@ logical operators, we are _lessening_ the burden of having to remember the `cfg`
 # Explanation
 [explanation]: #explanation
 [cfg-syntax]: https://doc.rust-lang.org/reference/conditional-compilation.html#r-cfg.syntax
+[precedence]: https://doc.rust-lang.org/reference/expressions.html#expression-precedence
 
 `#[cfg(foo && bar)]` enables the annotated code if and only if both `foo` **and** `bar` are enabled.
 Similarly, `#[cfg(foo || bar)]` enables the annotated code if and only if either `foo` **or** `bar`
 is enabled. Finally, `#[cfg(!foo)]` enables the annotated code if and only if `foo` is **not**
-enabled. `#[cfg_attr]` and `cfg!()` behave the same way. Precedence is the same as in expressions.
-
-In terms of formal syntax, the [`[cfg.syntax]`][cfg-syntax] is changed to the following:
+enabled. `#[cfg_attr]` and `cfg!()` behave the same way.
+
+Precedence is the [same as in expressions][precedence].
```

Comment:
you'll want to note `=` is treated as `==` for precedence purposes, since assignment has much lower precedence.

Reply:
done